src/main/java/lotto/controller/LottoController.java

```diff
@@ -19,9 +19,12 @@
 
     public void run() {
         Money money = Money.of(InputView.requestMoney());
-        LottoTickets lottoTickets = LottoTickets
-                .buy(new RandomNumberGenerator(LottoNumber.MIN_LOTTO_NUMBER, LottoNumber.MAX_LOTTO_NUMBER), money);
-        OutputView.outputTickets(lottoTickets);
+        ManualBuyCount manualBuyCount = ManualBuyCount.of(InputView.requestManualBuyCount());
+
+        LottoTickets lottoTickets = LottoTickets.buyManualTickets(InputView.requestManualTickets(manualBuyCount), money);
+        lottoTickets.buyAutoTickets(new RandomNumberGenerator(LottoNumber.MIN_LOTTO_NUMBER, LottoNumber.MAX_LOTTO_NUMBER), money);
```

Comment:
```suggestion
        LottoTickets manualLottoTickets = LottoTickets.buyManualTickets(InputView.requestManualTickets(manualBuyCount), money);
        LottoTickets LottoTickets= manualLottoTickets.buyAutoTickets(new RandomNumberGenerator(LottoNumber.MIN_LOTTO_NUMBER, LottoNumber.MAX_LOTTO_NUMBER), money);
```
이런 플로우로 작성하면 불변을 유지할 수 있을 것 같아요 🙂

Reply:
오.. 이런 방법도 있군요. 뭔가 `manualLottoTickets.buyAutotickets`라는 부분이 티켓이 티켓을 산다는게 아직 객체지향적 관점에서 익숙하지 않아서 그런지 떠오르지 않았네요. 감사합니다!